### PR TITLE
"Fix langchain version typo (#280)"

### PR DESCRIPTION
**Pull request by roshanr11**
## 📥 Pull Request

**📘 Description**
Changed langchain version from ~=1.19 to ==0.1.19.

**🔄 Related Issue (if applicable)**
#280 

**🎯 Goal**
To address the installation issue with agentops[langchain].

### CODE REVIEW

**Comment by braelyn-ai** [comment]
I was surprised we hadn't caught this, but it's only used for a couple langchain examples in our docs.
Can you change this to 0.2.14? Then I can merge it

File: pyproject.toml [hunk 1]
```diff
@@ -33,7 +33,7 @@ dev = [
     "tach~=0.9",
 ]
 langchain = [
-    "langchain~=1.19"
+    "langchain==0.1.19"
```
Comment (braelyn-ai) [inline pyproject.toml]: ==0.2.14

**Review by braelyn-ai — DISMISSED** [review]

**Review by roshanr11 — COMMENTED** [review]
updated!

**Review by roshanr11 — COMMENTED** [review]
lgtm

**Comment by HowieG** [comment]
Thanks @roshanr11 !

**Review by HowieG — APPROVED** [review]

**Pull request merged**